### PR TITLE
Run mochitests

### PR DESCRIPTION
**Pull request by juliandescottes**
I thought we updated the exports necessary for mc, but looks like I was wrong.

Also I don't think we use the root index.js so I deleted it.

cc @jasonLaster

### CODE REVIEW

**Comment by juliandescottes** [comment]
This work needs to wait for https://bugzilla.mozilla.org/show_bug.cgi?id=1325401 to land

File: index.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-const Rep = require("./src/reps/rep").Rep;
```
Comment (jasonLaster) [inline index.js]: 🔫

File: src/reps/load-reps.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
```
Comment (jasonLaster) [inline src/reps/load-reps.js]: 👍

**Comment by jasonLaster** [comment]
We'll likely want to update the MC docker image soon so that we pull in the reps changes which will be important for this to work

**Comment by juliandescottes** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1325401 just landed in central. 

I just resynchronized all reps with what we have in MC too.

**Comment by juliandescottes** [comment]
This is almost ready to land, but we first have to merge https://github.com/devtools-html/devtools-core/pull/90, release a new version of launchpad, and use it here.

**Comment by juliandescottes** [comment]
@jasonLaster I would be in favor of merging this before trying to update the docker image.

**Comment by jasonLaster** [comment]
I'll review soon and then we can land. 

The docker image should be easy to update

**Comment by jasonLaster** [comment]
I squashed the commits into one so that it would be easier to take out if there is something we need to change.

I hope to have time to setup the docker image this weekend so we can try running the mochitests on circle monday.

**Pull request merged**